orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py

```diff
@@ -570,6 +568,8 @@ def test_model_and_input_without_device():
     out is not None
 
 def test_model_with_different_devices_same_session():
+    os.environ['ORTMODULE_SKIPCHECK_POLICY'] = 'SKIP_CHECK_DISABLED'
```

Comment:
why we need this env variable for running the test? 
Would test fails without this env var?

Reply:
Yes, in the case of device change, UTs would fail wo/ disabling skipchecks.